testing/src/main/scala/compstak/kafkastreams4s/testing/KafkaStreamsTestRunner.scala

```diff
@@ -85,7 +90,7 @@ object KafkaStreamsTestRunner {
 
   def props: ju.Properties = {
     val p = new ju.Properties
-    p.put(StreamsConfig.APPLICATION_ID_CONFIG, "kafkastreams4s-test")
+    p.put(StreamsConfig.APPLICATION_ID_CONFIG, UUID.randomUUID().toString)
```

Comment:
Can we lift this whole thing into `F` and then flatMap it as needed? :)

Reply:
Yeah totally, doh.

Reply:
Let me know if https://github.com/compstak/KafkaStreams4s/pull/39/commits/2081792c60dd2736737f92c5189b95966569d05f was what you were thinking.